Index: org/jivesoftware/smackx/pubsub/Node.java
===================================================================
--- org/jivesoftware/smackx/pubsub/Node.java	(revision 11464)
+++ org/jivesoftware/smackx/pubsub/Node.java	(working copy)
@@ -48,7 +48,7 @@
 	 * @param connection The connection the node is associated with
 	 * @param nodeName The node id
 	 */
-	Node(Connection connection, String nodeName)
+	protected Node(Connection connection, String nodeName)
 	{
 		con = connection;
 		id = nodeName;
