Index: org/jivesoftware/smackx/pubsub/PubSubElementType.java
===================================================================
--- org/jivesoftware/smackx/pubsub/PubSubElementType.java	(revision 11464)
+++ org/jivesoftware/smackx/pubsub/PubSubElementType.java	(working copy)
@@ -41,6 +41,7 @@
 	SUBSCRIBE("subscribe", PubSubNamespace.BASIC), 
 	SUBSCRIPTION("subscription", PubSubNamespace.BASIC),
 	SUBSCRIPTIONS("subscriptions", PubSubNamespace.BASIC), 
+	SUBSCRIPTIONS_OWNER("subscriptions", PubSubNamespace.OWNER),
 	UNSUBSCRIBE("unsubscribe", PubSubNamespace.BASIC); 
 
 	private String eName;
Index: org/jivesoftware/smackx/pubsub/Node.java
===================================================================
--- org/jivesoftware/smackx/pubsub/Node.java	(revision 11464)
+++ org/jivesoftware/smackx/pubsub/Node.java	(working copy)
@@ -131,6 +131,22 @@
 	}
 
 	/**
+	 * Get all subscriptions currently associated with this node.
+	 * This includes foreign jids and uses the #owner namespace.
+	 *
+	 * @return List of {@link Subscription}
+	 *
+	 * @trhows XMPPException
+	 */
+	public List<Subscription> getAllSubscriptions()
+		throws XMPPException
+	{
+		PubSub reply = (PubSub)sendPubsubPacket(Type.GET, new NodeExtension(PubSubElementType.SUBSCRIPTIONS_OWNER, getId()), PubSubNamespace.OWNER);
+		SubscriptionsExtension subElem = (SubscriptionsExtension)reply.getExtension(PubSubElementType.SUBSCRIPTIONS_OWNER);
+		return subElem.getSubscriptions();
+	}
+
+	/**
 	 * The user subscribes to the node using the supplied jid.  The
 	 * bare jid portion of this one must match the jid for the connection.
 	 * 
